packages/react-vapor/src/components/toast/Toast.tsx

```diff
@@ -67,6 +69,23 @@ export class Toast extends React.Component<IToastProps> {
         }
     }
 
+    private getInfoToken(type: string) {
+        let infoToken = '';
+        if (type === ToastType.Success || (_.isEmpty(this.props.className) && _.isEmpty(this.props.type))) {
+            infoToken = 'info-token-full-success-32';
+        }
+        if (type === ToastType.Info) {
+            infoToken = 'info-token-full-information-32';
+        }
+        if (type === ToastType.Warning) {
+            infoToken = 'info-token-full-warning-32';
+        }
+        if (type === ToastType.Error) {
+            infoToken = 'info-token-full-critical-32';
+        }
+        return infoToken;
```

Comment:
We could also directly use `return` for each case instead of setting the variable `infoToken`.